db/Cargo.toml

```diff
@@ -7,6 +7,9 @@ edition = "2021"
 
 [dependencies]
 color-eyre = { workspace = true }
+create = "0.1.0"
+migrate = "0.2.0"
+page = "0.0.0"
```

Comment:
I think these might be things you accidentally added with `cargo add` maybe?

I think we can remove em!